test/parallel/test-buffer-fill.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
+const errors = require('../../lib/internal/errors');
```

Comment:
Can this be `require('internal/errors');` along with a `// Flags: --expose-internals` comment at the top of the file instead?